arduino:
  peripheral:
    ftdi: B0004Z2N
    fqbn: arduino:avr:nano
    core: arduino:avr
    sketch: sketchbooks/PeripheralArduino
    libraries:
      - servo
    sensors:
      - type: pressure
        tag: P
        topic: /sensors/depth
      - type: voltage
        tag: V
        topic: /sensors/voltage
      - type: temperature
        tag: T
        topic: /sensors/temperature
      - type: humidity
        tag: H
        topic: /sensors/humidity
    servos:
      - name: marker dropper
        tag: M
        min_pwm: 1100
        max_pwm: 1900
      - name: torpedoes
        tag: T
        min_pwm: 1100
        max_pwm: 1900
  thruster:
    ftdi: B0004VDI
    fqbn: arduino:avr:nano
    core: arduino:avr
    sketch: sketchbooks/ThrusterArduino